### PR TITLE
[bug fix][web] previous and next nav button issue fix

### PR DESCRIPTION
**Pull request by ann-aot**
-previous and next nav button was not working in dev, as the `formInstance` used in vue template was not reactive
**bug**
image

**fix**
image
image

### CODE REVIEW

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by dheepak-aot — APPROVED** [review]
👍 Nice catch.

**Review by henryoforeh-aot — APPROVED** [review]

File: sources/packages/web/src/components/common/StudentApplication.vue [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default {
     const PROGRAM_NOT_LISTED = "myProgramNotListed";
     const OFFERING_NOT_LISTED = "myStudyPeriodIsntListed";
 
-    let formInstance: any;
+    const formInstance = ref();
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/common/StudentApplication.vue:82]: Please consider keeping the _formInstance_ as a regular property and creating a computed property to deal with the visibility of the navigation buttons.
The _formInstance_ is not actually being used in the template and it is usually not needed.

**Review by andrewsignori-aot — COMMENTED** [review]
Nice catch but please take a look at the comment.

**Review by andrewsignori-aot — APPROVED** [review]
Looks good! Thanks for doing the change 👍

**Pull request merged**